sdk/tvOS/OoyalaTVSkinSDK/OOTVOptionsCollectionViewController.m

```diff
@@ -0,0 +1,133 @@
+    [self.optionsCollectionView setDataSource:self];
+    [self.optionsCollectionView setDelegate:self];
+    
+    self.optionsCollectionView.userInteractionEnabled = YES;
+    
+    [self.optionsCollectionView registerClass:[OOOoyalaTVOptionCell class] forCellWithReuseIdentifier:@"cellIdentifier"];
+    
+    //Setting default selected language to Off
+    self.selectedLanguage = @"Off0";
+    
+    [self.view addSubview:self.barView];
+    [self.view addSubview:self.optionsCollectionView];
+    [self.view bringSubviewToFront:self.optionsCollectionView];
+    
+    return self;
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+}
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section {
+    self.optionList = [[NSMutableArray alloc] initWithArray:[self.viewController availableOptions]];
+    Pair *sectionPair = [[Pair alloc] init];
+    sectionPair = [self.optionList objectAtIndex:section];
+    
+    return sectionPair.value.count;
+}
+
+- (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView {
+    //We are only supporting one section for CC menu
+    return 1;
+}
+
+- (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cellForItemAtIndexPath:(NSIndexPath *)indexPath {
+    OOOoyalaTVOptionCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:@"cellIdentifier" forIndexPath:indexPath];
+    
+    //We retrieve array with CC lenguages available to display corresponding cells
+    self.optionList = [[NSMutableArray alloc] initWithArray:[self.viewController availableOptions]];
+    
+    // Set the data for this cell:
+    Pair *sectionPair = [[Pair alloc] init];
+    sectionPair = [self.optionList objectAtIndex:indexPath.section];
+    cell.optionLabel.text = [sectionPair.value objectAtIndex:indexPath.row];
+    
+    //We display check if cell was already selected
+    if (self.selectedLanguage != nil && [self.selectedLanguage isEqualToString:[cell.optionLabel.text stringByAppendingFormat:@"%d",indexPath.row]]){
+        [cell.checkedLabel setHidden:NO];
+    }else {
+        [cell.checkedLabel setHidden:YES];
+    }
+    
+    return cell;
+}
+
+- (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollectionViewLayout *)collectionViewLayout sizeForItemAtIndexPath:(NSIndexPath *)indexPath {
+    return CGSizeMake(labelHeight * 26, labelHeight * 4);
+}
+
+- (BOOL)collectionView:(UICollectionView *)collectionView canFocusItemAtIndexPath:(NSIndexPath *)indexPath {
+    return YES;
+}
+
+#pragma mark - UICollectionViewDelegate
+- (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPath:(NSIndexPath *)indexPath {
+    OOOoyalaTVOptionCell *cell = [collectionView cellForItemAtIndexPath:indexPath];
```

Comment:
Cast to `OOOoyalaTVOptionCell *` to remove warnings in Xcode. Do the same in all the places that apply